web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknadsbehandling/LeggTilGrunnlagFormueRoutes.kt

```diff
@@ -0,0 +1,141 @@
+package no.nav.su.se.bakover.web.routes.søknadsbehandling
+
+import arrow.core.Either
+import arrow.core.NonEmptyList
+import arrow.core.getOrHandle
+import arrow.core.left
+import arrow.core.right
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.post
+import no.nav.su.se.bakover.common.periode.PeriodeJson
+import no.nav.su.se.bakover.common.serialize
+import no.nav.su.se.bakover.domain.Brukerrolle
+import no.nav.su.se.bakover.domain.grunnlag.Formuegrunnlag
+import no.nav.su.se.bakover.domain.grunnlag.KunneIkkeLageFormueVerdier
+import no.nav.su.se.bakover.domain.satser.SatsFactory
+import no.nav.su.se.bakover.domain.søknadsbehandling.Søknadsbehandling
+import no.nav.su.se.bakover.service.søknadsbehandling.SøknadsbehandlingService
+import no.nav.su.se.bakover.service.søknadsbehandling.SøknadsbehandlingService.KunneIkkeLeggeTilFormuegrunnlag
+import no.nav.su.se.bakover.service.søknadsbehandling.SøknadsbehandlingService.KunneIkkeLeggeTilFormuegrunnlag.FantIkkeSøknadsbehandling
+import no.nav.su.se.bakover.service.vilkår.LeggTilFormuegrunnlagRequest
+import no.nav.su.se.bakover.web.Resultat
+import no.nav.su.se.bakover.web.errorJson
+import no.nav.su.se.bakover.web.features.authorize
+import no.nav.su.se.bakover.web.routes.Feilresponser
+import no.nav.su.se.bakover.web.routes.Feilresponser.depositumErHøyereEnnInnskudd
+import no.nav.su.se.bakover.web.routes.grunnlag.FormuegrunnlagJson
+import no.nav.su.se.bakover.web.routes.grunnlag.tilResultat
+import no.nav.su.se.bakover.web.routes.periode.toPeriodeOrResultat
+import no.nav.su.se.bakover.web.routes.søknadsbehandling.FormueBody.Companion.toServiceRequest
+import no.nav.su.se.bakover.web.sikkerlogg
+import no.nav.su.se.bakover.web.svar
+import no.nav.su.se.bakover.web.withBehandlingId
+import no.nav.su.se.bakover.web.withBody
+import no.nav.su.se.bakover.web.withSakId
+import java.util.UUID
+
+private data class FormueBody(
```

Comment:
Regner med det eksisterer en type/mapping her som er felles for både revurdering og søknadsbehandling?